cli/azd/pkg/infra/provisioning/bicep/testdata/failed-subscription-deployment.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
This was the result of `az deployment sub show` for the failed deployment I got by running the script provided in #2508.